Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:latest
```

Comment:
create a separate Directory for docker and move this file there.

Reply:
++